binaries/ofs.uio/uio/ofs_uio.py

```diff
@@ -70,6 +72,48 @@ def verify_pcie_address(pcie_address):
     return True
 
 
+def verify_hex(str):
+    regx = re.compile("0[xX][0-9a-fA-F]+")
+    try:
+        if re.fullmatch(regx, str) is not None:
+            return True
+        else:
+            return False
+    except ValueError:
+        return False
+
+
+def verify_json(str):
```

Comment:
str is a Python built-in.  https://docs.python.org/3/library/functions.html Using it as a variable name is confusing.

Reply:
fixed